### ISSUE TITLE
__main__.py

### ISSUE THREAD

**Issue by 0815Unbekannt** [BillSchumacher/Auto-Vicuna#6]
on the file __main__.py they ask for     device, conversation_from_template("vicuna_v1.1"),
in fastchat.conversation.py you find this

default_conversation = conv_v1_2
conv_templates = {
    "v1": conv_v1_2,
    "bair_v1": conv_bair_v1,
}

__main__.py (row 90) 
search for               device, conversation_from_template("vicuna_v1.1"),
change from "vicuna_v1.1" to "v1"
its looks like           device, conversation_from_template("v1"),

**Comment by BillSchumacher** [comment]
Yeah this needs to be updated for the lastest version of FastChat, I think this package is obsoleted by the latest version as well actually.

I have an example of using their API server in a standalone manner which was the entire goal of this project:

https://github.com/BillSchumacher/Athena/blob/main/docker-compose.yml

**Comment by BillSchumacher** [comment]
Which was actually merged to upstream FastChat already as well I believe.

**Comment by BillSchumacher** [comment]
https://github.com/lm-sys/FastChat/blob/main/docker/docker-compose.yml
